### ISSUE TITLE
Adjust alert box colors to improve accessibility

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1497]
@aschmoe documenting here for us to review the alert boxes CSS. 

```
.alert-info {
	background-color: #f1f1f1;
    border-color: #f1f1f1;
    color: #212121;
}
```

**Comment by lukefretwell** [comment]
@kevindherman here's the issue for the alert bar. @aschmoe added accessibility label to this as we need to review and update the coloring.

**Comment by lukefretwell** [comment]
Actually, this might be related: https://github.com/proudcity/wp-proudcity/issues/1393

**Comment by kevindherman** [comment]
@lukefretwell I moved this to in progress and up to the top of the list so it doesn't get lost.

**Comment by jlyon** [comment]
```
// Init styles
// ---------------------------

@import "loaders/load-local";

// Local Styles
// ---------------------------

@import "common/global";
@import "components/buttons";
@import "components/comments";
@import "components/forms";
@import "components/grid";
@import "components/type";
@import "components/wp-classes";
@import "layouts/header";
@import "layouts/sidebar";
@import "layouts/footer";
@import "layouts/pages";
@import "layouts/posts";
@import "layouts/tinymce";

@import "plugins/events-manager";
@import "plugins/job_manager";
@import "plugins/gravity-forms";

// @import "modules/wr-pagebuilder";



$proud-alert-red: #BF0711;
$proud-alert-orange: #C05717;
$proud-alert-yellow: #EEC200;
$proud-alert-green: #108043;
$proud-alert-black: #212B35;
$proud-alert-gray: #C4CDD5;


.proud-alert-banner {
  border-radius: 0 !important;
  border: none !important;
}

.alert-red {
  background-color: $proud-alert-red;
}

.alert-orange {
  background-color: $proud-alert-orange;
}

.alert-yellow {
  background-color: $proud-alert-yellow;
}

.alert-green {
  background-color: $proud-alert-green;
}

.alert-black {
  background-color: $proud-alert-black;
}

.alert-gray {
  background-color: $proud-alert-gray;
}

.alert-red,
.alert-orange,
.alert-green,
.alert-black {
  color: $white;
  
  a {
    color: $white !important;
    text-decoration: underline;
  }
}
```

**Comment by aschmoe** [comment]
@lukefretwell @kevindherman I just review these colors and all are accessible.  If you can point me to a site where this has been a problem I'm happy to take a look, or if we don't like any of the colors, we can change them but from an accessibility standpoint we're good.

https://webaim.org/resources/contrastchecker/

**Comment by kevindherman** [comment]
@aschmoe I believe @lukefretwell changed the CSS on city template but not the platform. Luke, could you confirm?

I think one of the challenges is that even if the font color contrast is ok there is a blue used for hyperlinks that doesn't contrast enough. I know this is definitely the case for the blue or sometimes the highlight color option.

**Comment by lukefretwell** [comment]
@aschmoe @kevindherman I'll take a look.

**Comment by kevindherman** [comment]
make link text same color as main text with underline.

**Comment by aschmoe** [comment]
@kevindherman @lukefretwell this is going out with release
